config/v1/types_feature.go

```diff
@@ -117,6 +117,7 @@ var FeatureSets = map[FeatureSet]*FeatureGateEnabledDisabled{
 		with("BuildCSIVolumes").             // sig-build, adkaplan, OCP specific
 		with("NodeSwap").                    // sig-node, ehashman, Kubernetes feature gate
 		with("MachineAPIProviderOpenStack"). // openstack, egarcia (#forum-openstack), OCP specific
+		with("InsightsAPI").                 // insights, tremes (#ccx), OCP specific
```

Comment:
Have you looked at generating the CRD yaml resource? You'll need this to then get it installed into the cluster. When you do so, you'll want to make sure the CRD resource has the appropriate annotation to make sure it's installed in the cluster only when the feature gate is enabled